selfdrive/ui/ui.cc

```diff
@@ -193,10 +194,15 @@ static void update_state(UIState *s) {
       }
     }
   }
-  if (Hardware::TICI() && sm.updated("roadCameraState")) {
+  if (sm.updated("roadCameraState")) {
     auto camera_state = sm["roadCameraState"].getRoadCameraState();
-    float gain = camera_state.getGainFrac() * (camera_state.getGlobalGain() > 100 ? 2.5 : 1.0) / 10.0;
-    scene.light_sensor = std::clamp<float>((1023.0 / 1757.0) * (1757.0 - camera_state.getIntegLines()) * (1.0 - gain), 0.0, 1023.0);
+    if (Hardware::EON()) {
```

Comment:
should we check the inverse of the condition for the light sensor? you'll get weird behavior here with driver view for example

Reply:
or better yet use different variables and handle this logic in the brightness code